src/dragonfly_reporter/observations.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+import httpx
+from fastapi.encoders import jsonable_encoder
+
+from dragonfly_reporter.models import Observation
+
+
+async def send_observation(
+    project_name: str, observation: Observation, *, http_client: httpx.AsyncClient
+):
+    url = f"/danger-api/projects/{project_name}/observations"
+    json = jsonable_encoder(observation)
+
+    await http_client.post(url, json=json)
```

Comment:
Technically not a url
```suggestion
    path = f"/danger-api/projects/{project_name}/observations"
    json = jsonable_encoder(observation)

    await http_client.post(path, json=json)
```